extensions/amp-social-share/1.0/social-share-config.js

```diff
@@ -14,19 +14,23 @@
  * limitations under the License.
  */
 
-import {dict} from '../../../src/utils/object';
-
 /**
  * Get social share configurations by supported type.
- * @param  {string} type
- * @return {!SocialShareConfigDef}
+ * @param  {?string} type
+ * @return {?SocialShareConfigDef}
```

Comment:
`SocialShareConfigDef|undefined`?

Reply:
Alternatively we can `return BUILTINS[type] || {}` and remove the `|| {}` from where this is called in `SocialShare`

Reply:
Thanks, updated!